benchmarking/blackbox_repository/blackbox_tabular.py

```diff
@@ -81,14 +81,19 @@ def __init__(
 
     def _objective_function(
             self,
-            configuration: Dict,
+            configuration: Union[Dict, int],
```

Comment:
This seems odd. Did you change this in the baseclass as well?

Reply:
This is to be able to support a comparison with LCbench which access configuration by integer, not by dictionary.